src/stories/Button.stories.js

```diff
@@ -0,0 +1,51 @@
+import Button from "./Button.svelte";
```

Comment:
The empty Button.svelte looks suspicious but not critical for the PR.

Reply:
Good point, removed it. There are enough other examples.